client/rest-high-level/src/test/java/org/elasticsearch/client/BulkProcessorIT.java

```diff
@@ -66,10 +81,10 @@ public void testThatBulkProcessorCountIsCorrect() throws Exception {
 
         int numDocs = randomIntBetween(10, 100);
         try (BulkProcessor processor = initBulkProcessorBuilder(listener)
-                //let's make sure that the bulk action limit trips, one single execution will index all the documents
-                .setConcurrentRequests(randomIntBetween(0, 1)).setBulkActions(numDocs)
-                .setFlushInterval(TimeValue.timeValueHours(24)).setBulkSize(new ByteSizeValue(1, ByteSizeUnit.GB))
-                .build()) {
```

Comment:
i see a lot of these in this file, if you are not modifying the line w/ a content change, pls dont let your ide format it :P

Reply:
will revert these formattings